app/tests/src/common/address-manager-settings.js

```diff
@@ -51,23 +51,28 @@ const addressManagerSettings = [
   },
 ]
 
-const addressManagerSettings2 = [
-  {
+// these are used for the new tests
+const addressManagerSettings2 = {
```

Comment:
"new" is a very relative term in the codebase - I'd rather name this `addressManagerSettings` and the "old" `addressManagerSettings` to `addressManagerSettings_deprecated` or something - if we really want to get rid of those. And if not, we should probably find appropriate name for both separately as they even have different fields.

In general, "settings" is quite an overloaded term and I find it more suitable to name it "fixtures" and split the "parameters" from "expected" results as currently it's really confusing having both mixed, but that's probably worth addressing separately.

Reply:
The comment was mostly for me tbh since I expected that if anyone, I ll be working on this further.. :D but I ll update the comment and fix the names. Regarding the `settings` I was only following the previous implementation and I agree, we can address that separately.

Reply:
I decided to remove the older Shelley tests altogether since they are testing the same stuff as the new ones anyway.